comp/logs/agent/config/endpoints.go

```diff
@@ -34,14 +35,15 @@ const (
 
 // Endpoint holds all the organization and network parameters to send logs to Datadog.
 type Endpoint struct {
-	APIKey                  string `mapstructure:"api_key" json:"api_key"`
-	Host                    string
+	apiKeyGetter func() string
+	isReliable   bool
+	useSSL       bool
+
+	Host                    string `mapstructure:"host" json:"host"`
```

Comment:
Just want to double check here that ``mapstructure:"host" json:"host"`` is case insensitive in how it's used. Because previously it was treated as `Host`. 

Since the tests are passing - I am assuming this is all good, but just wanted to double check.

Reply:
Yes, this config still works:
```
  additional_endpoints:
  - api_key: ENC[api_key_2]
    Host: agent-http-intake.logs.datadoghq.com
    is_reliable: true
```